src/main/java/frc/robot/RobotContainer.java

```diff
@@ -119,6 +124,15 @@ public void traj() {
     SmartDashboard.putData("field", drive.field);
   }
 
+  public int isCube(boolean isCube) {
```

Comment:
Separate this into commands (can be inline)

Reply:
Anish says it's fine (if it works)

Reply:
Meh, let's make it work now and stick it in a command